### PR TITLE
Visualizer

### PR DESCRIPTION
**Pull request by reshadi**
Initial setup and a sample component for the new visualizer package.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -15,6 +15,9 @@
     "**/node_modules": true,
     "**/*.js": {
       "when": "$(basename).ts"
+    },
+    "**/*.jsx": {
```
Comment (schwartzmx) [inline .vscode/settings.json:19]: Ah this is maybe what I was having trouble with when trying to set up the package and import tsx component into test app.

**Review by schwartzmx — APPROVED** [review]
thanks for setting this up,  we can merge and I can port what I was working on into this

File: .vscode/settings.json (hunk 1 above)
Comment (reshadi) [reply .vscode/settings.json:19]: I doubt that. This setting is to tell VS code don't show the generated JSX files if you find a file with the same basename but with TSX extension. 
I think your problem might have been that `jsx: react` option in the `tsconfig.json`

Comment (schwartzmx) [reply .vscode/settings.json:19]: Ah I see,  I had `"jsx": "react-jsx"`,  and thought I tried the other variations as well,  but maybe some other misconfiguration on my side

**Pull request merged**